docs/guide/db-active-record.md

```diff
@@ -1418,19 +1418,19 @@ Then you need to compose a query, which calculates volume of the room and perfor
 $rooms = Room::find()
     ->select([
         '{{room}}.*', // select all columns
-        '([[length]] * [[width]].* [[height]]) AS volume', // calculate a volume
+        '([[length]] * [[width]] * [[height]]) AS volume', // calculate a volume
     ])
     ->orderBy('volume DESC') // apply sort
     ->all();
 
 foreach ($rooms as $room) {
-    echo $room->volume; // contains value calculated by SQL
+    echo $room->volume, '<br>'; // contains value calculated by SQL
```

Comment:
I don't think this example is correct in terms of responsibility.

If this is code from view - then `Room::find()` is wrong, because views must not be responsible for data retentive from the database.
If this code is controller - then `<br>` is wrong, because controllers must not render HTML.